tests/ui/UnreadIndicatorsTest.js

```diff
@@ -0,0 +1,395 @@
+/* eslint-disable @lwc/lwc/no-async-await */
+import React from 'react';
+import Onyx from 'react-native-onyx';
+import {Linking, AppState} from 'react-native';
+import {fireEvent, render, act} from '@testing-library/react-native';
+import lodashGet from 'lodash/get';
+import moment from 'moment';
+import App from '../../src/App';
+import CONST from '../../src/CONST';
+import ONYXKEYS from '../../src/ONYXKEYS';
+import waitForPromisesToResolve from '../utils/waitForPromisesToResolve';
+import * as TestHelper from '../utils/TestHelper';
+import appSetup from '../../src/setup';
+import fontWeightBold from '../../src/styles/fontWeight/bold';
+import * as AppActions from '../../src/libs/actions/App';
+import ReportHeaderViewBackButton from '../../src/pages/home/ReportHeaderViewBackButton';
+import ReportActionsView from '../../src/pages/home/report/ReportActionsView';
+import * as NumberUtils from '../../src/libs/NumberUtils';
+import LocalNotification from '../../src/libs/Notification/LocalNotification';
+import * as Report from '../../src/libs/actions/Report';
+
+beforeAll(() => {
+    global.fetch = TestHelper.getGlobalFetchMock();
+
+    // We need a bit more time for this test in some places
+    jest.setTimeout(30000);
+    Linking.setInitialURL('https://new.expensify.com/r/1');
+    appSetup();
+});
+
+/**
+ * @param {RenderAPI} renderedApp
+ */
+function scrollUpToRevealNewMessagesBadge(renderedApp) {
+    fireEvent.scroll(renderedApp.getByTestId('report-actions-list'), {
+        nativeEvent: {
+            contentOffset: {
+                y: 250,
+            },
+            contentSize: {
+                // Dimensions of the scrollable content
+                height: 500,
+                width: 100,
+            },
+            layoutMeasurement: {
+                // Dimensions of the device
+                height: 700,
+                width: 300,
+            },
+        },
+    });
+
+    // We advance the timer since we must wait for the animation to end
+    // and the new style to be reflected
+    jest.advanceTimersByTime(100);
+}
+
+/**
+ * @param {RenderAPI} renderedApp
+ * @return {Boolean}
+ */
+function isNewMessagesBadgeVisible(renderedApp) {
+    const badge = renderedApp.getByTestId('new-messages-badge');
+    return badge.props.style.transform[0].translateY === 10;
+}
+
+/**
+ * @param {RenderAPI} renderedApp
+ * @return {Promise}
+ */
+async function navigateToSidebar(renderedApp) {
```

Comment:
`await` is not used inside this function so I think we can remove the `async` here

```suggestion
function navigateToSidebar(renderedApp) {
```